examples/cmp-vendors.amp.html

```diff
@@ -163,6 +163,31 @@ <h3>Image that is NOT blocked by consent</h3>
         </amp-consent>
         <!-- End didomi example -->
 
+      <!-- SourcePoint example -->
+      <amp-consent id='consent' layout='nodisplay' type='SourcePoint'>
+        <script type="application/json">
+          {
+            "consents": {},
+            "postPromptUI": "my-consent-prompt-ui",
+            "clientConfig": {
+              "accountId": 22,
+              "siteName": "mobile.demo",
+              "siteId": 2372,
+              "privacyManagerId": "5c0e81b7d74b3c30c6852301",
```

Comment:
Just to make sure this is created for example usage and is ok to be shared?

Reply:
Good point. All those attributes were created and they are ok to be shared. I've, however, changed to another config specifically for demoing our AMP solution.